test_integration/test_frequency_hint_usage.py

```diff
@@ -98,13 +99,13 @@ def setUpClass(cls):
 
         cls._fmap_report_path = 'test_{}_fmap.report'.format(test_name)
         cls._fmap_trace_path = 'test_{}_fmap.trace'.format(test_name)
-        cls._fmap_agent_conf_path = test_name + '-fmap-agent-config.json'
+        cls._fmap_agent_conf_path = 'test_' + test_name + '-fmap-agent-config.json'
 
         cls._ee_report_path = 'test_{}_ee.report'.format(test_name)
         cls._ee_trace_path = 'test_{}_ee.trace'.format(test_name)
-        cls._ee_agent_conf_path = test_name + '-ee-agent-config.json'
+        cls._ee_agent_conf_path = 'test_' + test_name + '-ee-agent-config.json'
 
-        geopmpy.error.exc_clear() # Clear out exception record for python 2 support
+        geopmpy.error.exc_clear()  # Clear out exception record for python 2 support
```

Comment:
Why the space?

Reply:
pretty sure this is a pep8 style guideline; my linter was complaining.